site/docs/windows.md

```diff
@@ -198,6 +198,54 @@ To build C++ targets, you need:
     set BAZEL_WINSDK_FULL_VERSION=10.0.10240.0
     ```
 
+*   The Clang compiler
+    
+    From 0.29.0, Bazel supports building with LLVM's MSVC-compatible compiler driver (`clang-cl.exe`).
+    To build with `clang-cl.exe`, you have to install **both** LLVM and Visual C++ Build tools.
+    Because we still need to link to Visual C++ libraries.
+
+    To install LLVM, please visit http://releases.llvm.org/download.html.
+    
+    Bazel can automatically detect LLVM installation on your system, you can explicitly tell Bazel where LLVM is installed by `BAZEL_LLVM`.
```

Comment:
Missing word? Did you mean:

Bazel can automatically detect LLVM installation on your system, **[or]** you can explicitly tell Bazel (...)

Reply:
Done